libs/fixtures/src/data/electionMinimalExhaustiveSample/ballot-package.zip.ts

```diff
@@ -0,0 +1,50 @@
+/* Generated by res-to-ts. DO NOT EDIT */
```

Comment:
Nice, I do think we should start just storing ballot packages for the elections in libs/fixtures for our own use